core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/seo/components/dot-edit-page-view-as-controller-seo/dot-edit-page-view-as-controller-seo.component.html

```diff
@@ -16,6 +16,14 @@
         (delete)="deletePersonalization($event)"
         (selected)="changePersonaHandler($event)"
     ></dot-persona-selector>
+
+    <p-checkbox
+        class="flex w-7 dot-edit__what-changed-button"
```

Comment:
Should we use this class `w-7`? If I'm not wrong, it's from Prime Flex but we can check if we have some variable for this